### ISSUE TITLE
Matrix question options have incorrect aria-label

### ISSUE THREAD

**Issue by jongrim** [surveyjs/survey-library#3540]
### Are you requesting a feature, reporting a bug or asking a question?

Reporting an accessibility bug

### What is the current behavior?

On matrix questions, options are given the incorrect `aria-label` value, and the span embedded in the label has the incorrect value.

### What is the expected behavior?

The generated `aria-label` and hidden span should have a value that matches the title of the column, not the name of the question

### How would you reproduce the current behavior (if this is a bug)?

See codesandbox. You can reproduce this by creating a matrix question using the Survey Designer

#### Provide the test code and the tested page URL (if applicable)

Tested page URL: [Codesandbox example](https://codesandbox.io/s/serverless-cookies-m8b4m?file=/src/index.js)

Test code

```
import ReactDOM from "react-dom";
import * as Survey from "survey-react";

Survey.StylesManager.applyTheme("modern");

var surveyJSON = {
  pages: [
    {
      name: "page1",
      elements: [
        {
          type: "matrix",
          name: "question1",
          columns: ["Column 1", "Column 2", "Column 3"],
          rows: ["Row 1", "Row 2"]
        }
      ]
    }
  ]
};

function sendDataToServer(survey) {
  //send Ajax request to your web server.
  alert("The results are:" + JSON.stringify(survey.data));
}

ReactDOM.render(
  <Survey.Survey json={surveyJSON} onComplete={sendDataToServer} />,
  document.getElementById("surveyContainer")
);


```

### Specify your

* browser: Brave
* browser version: 1.30.89 - Chromium 94.0.4606.81
* surveyjs platform (angular or react or jquery or knockout or vue): example is in react, but have observed the issue in vue as well
* surveyjs version: 1.8.72
